### PR TITLE
Port subjects page to ts

### PR DESCRIPTION
**Pull request by RoyEJohnson**
[CORE-537]
This covers the all-subjects page. The specific-subjects page wll be a separate PR.

[CORE-537]: https://openstax.atlassian.net/browse/CORE-537?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/pages/subjects/new/specific/context.d.ts [hunk 1]
```diff
@@ -46,20 +48,14 @@ type WebinarSectionInfo = {
...
-
 type SpecificSubjectPageData = {
     translations?: [LocaleEntry[]];
     title?: string;
     subjects?: SubjectEntry;
     tutorAd: SectionInfo;
     aboutOs: SectionInfo;
     webinarHeader: WebinarSectionInfo;
-    infoBoxes: [InfoBox[]];
+    infoBoxes: InfoBox;
```
Comment (jivey) [inline src/app/pages/subjects/new/specific/context.d.ts:58]: Should this become infoBox?

**Review by jivey — COMMENTED** [review]
Is there need to keep the Tutor stuff around? Or would it break a bunch of stuff to remove it.

**Review by jivey — APPROVED** [review]
Wondering if we should pull out the Tutor stuff but otherwise looks good

**Comment by RoyEJohnson** [comment]
The tutor ad stuff has been repurposed to promote something other than Tutor. Ideally, it would be renamed.

**Pull request merged**